environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -35,6 +35,13 @@ objects:
             name: grpc
           - containerPort: 9090
             name: http
+          resources:
```

Comment:
Looks like the current configuration hardcoded CPU requests and limits to the querier container, and parameterized ones to the proxy in front of the querier.

Reply:
Good catch. Updating.